Automattic/jetpack
# Boost: Move Critical CSS Status styles into their own module

# Merge after https://github.com/Automattic/jetpack/pull/35104.

Part of https://github.com/Automattic/jetpack/issues/35010.

## Proposed changes:

* Make styles local, by moving them into their own component.
* Also add some missing spacing between elements. Mobile spacing has been adjusted as well.

<details><summary>Spacing before</summary>

<img src="https://github.com/Automattic/jetpack/assets/11799079/41d79b5e-f63b-4d81-a74a-33adc2db5025" />

</details>

<details><summary>Spacing now</summary>

<img src="https://github.com/Automattic/jetpack/assets/11799079/0ea4e3b8-cca1-4967-a30a-f0d31aa2ee30" />

</details>

### Other information:


## Jetpack product discussion



https://github.com/Automattic/jetpack/issues/35010

## Does this pull request change what data or activity we track or use?



no

## Testing instructions:






* Setup Boost;
* Make sure Critical CSS status section looks okay.
